src/components/ListPullRequests.vue

```diff
@@ -30,5 +35,73 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+  .list-pull-request {
+    padding: 0 10px;
+  }
+  .pr-title {
+    font-size: 20px;
+    font-weight: bold;
+    color: #0A5373;
+    margin: 15px 0;
+  }
+
+  .pr-item {
+    border-radius: 3px;
+    padding: 5px;
+    margin: 10px 0;
+    background-color: rgba(255,184,86, .2);
+  }
+
+  .pr-username {
+    color: #A64C44;
+    font-weight: bold;
+    font-size: 17px;
+  }
+
+  .pr-link {
+    width: 100%;
+    text-overflow: ellipsis;
+    display: block;
+    overflow: hidden;
+    text-decoration: none;
+    color: #09213E;
+
+    &:hover {
+      color: #F26B5E;
+    }
+  }
+
+  @media screen and (min-width: 768px) {
+    .list-pull-request {
+      text-align: center;
+    }
+
+    .pr-title {
+      font-size: 28px;
+    }
+    .pr-list {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: center;
 
+      .pr-item {
+        border-radius: 10px;
+        width: 25%;
```

Comment:
Please, let's make it always fill the whole width:

![image](https://user-images.githubusercontent.com/1295197/67123075-a7f1be00-f1c5-11e9-8942-b47c344d8643.png)

Since it is a ranking list, won't be much helpful placing things side by side.

Reply:
Done !